com.biglybt.BiglyBT.metainfo.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.biglybt.BiglyBT</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>BiglyBT</name>
+  <summary>Feature filled, open source, ad-free, bittorrent client</summary>
+  <description>
+    <p>Since 2003, BiglyBT's team has worked hard to bring you the best open source torrent product in the market. Unlike our competitors, our client is ad-free and our installer contains no third party offers.
+      We provide migration tools for several popular bittorent clients. Switching to BiglyBT from projects using the same source is simple and requires no interactions beyond the initial migration approval. Your settings, downloads, and plugins will all be there and ready to use.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Main window</caption>
+      <image>https://www.biglybt.com/img/ss/MainWindow.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Open Torrents</caption>
+      <image>https://www.biglybt.com/img/ss/OpenTorrents.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Search</caption>
+      <image>https://www.biglybt.com/img/ss/Search.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Dashboard View</caption>
+      <image>https://www.biglybt.com/img/ss/BiglyBTDesktop.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>File Tree</caption>
+      <image>https://www.biglybt.com/img/ss/BiglyBTFileTree.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Country Stats</caption>
+      <image>https://www.biglybt.com/img/ss/BiglyBTCountryStats.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>I2P Peers</caption>
+      <image>https://www.biglybt.com/img/ss/I2P_Peers.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Trackers View</caption>
+      <image>https://www.biglybt.com/img/ss/AllTrackers.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Colored Library</caption>
+      <image>https://www.biglybt.com/img/ss/ColoredLibrary.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://netbeans.apache.org/</url>
```

Comment:
I don't think this is the right URL for the homepage

Reply:
Correct, updated it